lib/govuk_index/elasticsearch_presenter.rb

```diff
@@ -0,0 +1,27 @@
+module GovukIndex
+  class ElasticsearchPresenter
+    def initialize(payload)
+      @payload = payload
+    end
+
+    def call
+      {
+        _type: payload["document_type"],
+        _id: payload["base_path"],
+        _source: {
+          link: payload["base_path"],
+          title: payload["title"]
+        }
+      }
+    end
+
+    def valid!
+      return if payload["base_path"]
+      raise(ValidationError, "base_path missing from payload")
```

Comment:
There's legitimate cases where valid content won't have base paths (world locations I think) - perhaps you want to just skip those instead of raising?